test/functional/helpers.lua

```diff
@@ -724,6 +724,10 @@ function module.pathroot()
   return iswin() and (module.nvim_dir:sub(1,2)..pathsep) or '/'
 end
 
+function module.getcwd()
+  return os.getenv("PWD")
```

Comment:
won't work on Windows (appveyor is failing), but not needed anyways. Could `lfs.currentdir()` instead, but for this case we can ask the Nvim process itself (we aren't actually interested in the CWD of the Lua test-runner process).

     eval('getcwd()')

Reply:
Sounds good to me - mind if I skip the `eval` and call `getcwd()` itself from the imported functions?

```
local getcwd = helpers.funcs.getcwd
```

Reply:
> Sounds good to me - mind if I skip the `eval` and call `getcwd()` itself from the imported functions?

Sure, eval() is just me being lazy :)